src/solvers/lowering/byte_operators.cpp

```diff
@@ -21,6 +21,46 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <util/simplify_expr.h>
 #include <util/string_constant.h>
 
+/// Determine the member of maximum fixed bit width in a union type. If no
+/// member, or no member of fixed and non-zero width can be found, return
+/// nullopt.
+/// \param union_expr: Expression to use as compound operand in the resulting
+///   member_exprt.
+/// \param union_type: Type to determine the member of.
+/// \param ns: Namespace to resolve tag types.
+/// \return Pair of a member_exprt pointing to the maximum fixed bit-width
+///   member of \p union_type and the bit width of that member.
+static optionalt<std::pair<member_exprt, mp_integer>>
+find_maximum_union_component(
```

Comment:
maximum -> widest?

Reply:
Fixed (and left-hand sides are renamed accordingly).